### DISCUSSION TITLE
SQLite Studio is frozen!! Help!!

### DISCUSSION THREAD

**Discussion by gwendolynflesher** [pawelsalawa/sqlitestudio#5064] [Q&A]
My SQLite Studio has been frozen for 4 days. It is giving me the error signal "[10:11:16] Contents of the SQL editor are huge,so errors detecting and existing objects highlighting are temporarily disabled". Has anyone had this happen to them before?? I have tried to uninstall and reinstall, restart my MacBook. Nothing has helped.

**Comment by pawelsalawa** [comment]
Delete (or rename) your configuration file: https://github.com/pawelsalawa/sqlitestudio/wiki/Configuration_directory_location

**Comment by gwendolynflesher** [comment]
I do not understand how to find the configuration file. I do not think I am techy enough yet.

**Reply by pawelsalawa** [reply]
On your Mac find application called Terminal, start it. There paste command:
```
rm -f $HOME/Library/Preferences/SalSoft/SQLiteStudio/settings3
```
and hit Enter key. You may close the Terminal app.

Now SQLiteStudio should run normally.

**Comment by gwendolynflesher** [comment]
Thank you!!!! I so appreciate your help!!
